changelogs/DP-17527.yml

```diff
@@ -0,0 +1,41 @@
+#
+# Write your changelog entry here. Every pull request must have a changelog yml file.
```

Comment:
Should the comments be removed for auto release?

Reply:
Good question.  We've had some issues recently, so I'll strip them out.